modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -23,4 +28,56 @@ $ oc patch networks.operator.openshift.io/cluster --type=json \
   -p='[{"op":"remove", "path":"/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig"}]'
 ----
 
+. To find the names of the OVN-Kubernetes data plane pods that exist on a node in your cluster, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-node
+----
++
+.Example output
+[source,terminal]
+----
+ovnkube-node-5xqbf                       8/8     Running   0              28m
+ovnkube-node-6mwcx                       8/8     Running   0              29m
+ovnkube-node-ck5fr                       8/8     Running   0              31m
+...
+----
+
+. To check a node in your cluster has IPsec disabled, enter the following command. Ensure that you enter this command for each node that exists in your cluster. The command output must state `false` to indicate that the node has IPsec disabled.
++
+[source,terminal]
+----
+$ oc -n openshift-ovn-kubernetes rsh ovnkube-node-<pod_number_sequence> ovn-nbctl --no-leader-only get nb_global . ipsec <1>
+----
+<1> Replace `<pod_number_sequence>` with the random sequence of letters, `5xqbf`, for a data plane pod from the previous step.
+
+. To remove the IPsec `ovn-ipsec-host` daemonset pod from the `openshift-ovn-kubernetes` namespace on a node, enter the following command:
++
+[source,terminal]
+----
+$ oc delete daemonset ovn-ipsec-host -n openshift-ovn-kubernetes <1>
+----
+<1> The `ovn-ipsec-host` daemonset pod configures IPsec connections for east-west traffic on a node.
+
+. To remove the IPsec `ovn-ipsec-containerized` daemonset pod from the `openshift-ovn-kubernetes` namespace on a node, enter the following command:
++
+[source,terminal]
+----
+$ oc delete daemonset ovn-ipsec-containerized -n openshift-ovn-kubernetes <1>
+----
+<1> The `ovn-ipsec-containerized` daemonset pod configures IPsec connections for east-west traffic on a node.
+
+. Verify that the `ovn-ipsec` and `ovn-ipsec-containerized` daemonset pods were removed from all the nodes in your cluster by entering the following command. If the command output does not list the pods, the removal operation is successful.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -l=app=ovn-ipsec
```

Comment:
@pperiyasamy. What about the `ovn-ipsec-containerized` daemonset pod here? Does this have the issue with the initial delete command?

Reply:
it covers both daemonset pods.